shell/platform/tizen/BUILD.gn

```diff
@@ -167,6 +170,10 @@ template("embedder_for_profile") {
       "//flutter/shell/platform/common/cpp/client_wrapper:client_wrapper",
       "//third_party/rapidjson",
     ]
+
+    if (enable_profiling) {
+      deps += [ "//flutter/runtime:libdart" ]
+    }
```

Comment:
Why does the embedder depend on the dart runtime? Shouldn't you just remove these lines?

Reply:
Ok, I've removed the dependency completely

Reply:
Thank you very much!